clients/web/test/testEnv.pug

```diff
@@ -14,7 +14,7 @@ html(lang="en")
     each js in jsFiles
       script(src=js)
     
-    script(src="../../../test/lib/promise.min.js")
```

Comment:
My bad, I saw the file in the diff list but it was collapsed so I didn't notice it was being removed instead of added.